Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java

```diff
@@ -105,35 +122,34 @@ public static void provideGeckoDriver(String geckoDriverVersion, String download
      */
     public static void provideChromeDriver(String downloadFolder, String extractFolder)
```

Comment:
Looks like a lot of duplicate code. Perhaps one could add a private function here and extract the shared code.

Reply:
Maybe, but since the two methods are sufficiently different to prevent simply moving some code to a new private method it is out of scope of this pull request, which just aims at re-enabling automatic builds.